### ISSUE TITLE
Fine-tuning on New Robot and New Task: Policy Appears to Only Learn Motions Without Successful Grasping

### ISSUE THREAD

**Issue by parkerpanjs** [NVIDIA/Isaac-GR00T#674]
### 🐛 Describe the bug

Description
We developed a custom data collection pipeline using a dexterous hand to perform cup grasping and placement tasks. After fine-tuning for 20,000 steps, the robot executes grasping and placing motions during deployment, but fails to actually lift the cup (see attached GIF/video).
Data Collection Setup
Robot: Dexterous hand
Task: Grasp a cup and place it into one of three holes
Dataset size: ~1,000 trajectories
Randomization:
Cup grasping position is randomized
Placement target is randomly selected among three holes with different colors
Action Representation
We use state offset by one frame as the action (i.e., action_t = state_{t+1} - state_t or directly using next-frame state as action).
Training Details
Training steps: 20,000 steps
Observation: (please specify if you use proprioception, visual input, or both)
Deployment Behavior
The robot performs grasping and placing motions sequentially
However, it fails to lift the cup — the fingers close but do not generate sufficient force/contact to pick up the object
See the attached GIF for visualization
Questions / Hypotheses
Could the policy be overfitting to motion trajectories without learning proper contact/force modulation?
Is the action representation (state delta) causing the policy to ignore gripper closure timing?
Would increasing dataset size or adding contact force observations help?
Attachments
[ https://github.com/user-attachments/assets/e4320305-d4a7-4b10-9981-15b4796eb09c

] GIF/video of deployment behavior
[ 

<img width="1500" height="1350" alt="Image" src="https://github.com/user-attachments/assets/e92b4ef3-0869-4b43-890f-b5c4b1a8aca3" />

] Training curves (loss, success rate if available)
[ 

<img width="800" height="12800" alt="Image" src="https://github.com/user-attachments/assets/fa3e0c18-4958-4cef-9454-5f9ff6b79a34" />

] MSE/joints
Would appreciate any guidance on:
Whether this is a known issue with state-delta action representations for contact-rich tasks
Recommended data volume for dexterous manipulation fine-tuning
Suggestions on action space design or reward engineering for grasp stability

### Versions

Isaaclab2.2.1 isaacsim5.0

**Comment by parkerpanjs** [comment]
#663 come with this issue
